files/default/handler/audit_report.rb

```diff
@@ -124,7 +124,13 @@ def get_opts(format, quiet)
 
       # run profiles and return report
       def call(opts, profiles)
-        Chef::Log.info "Initialize InSpec #{::Inspec::VERSION}"
+        Chef::Log.info "Using InSpec #{::Inspec::VERSION}"
+
+        if Gem::Version.new(Inspec::VERSION) < Gem::Version.new('1.24.0')
+          Chef::Log.error "This audit cookbook version requires InSpec 1.24.0 or newer, aborting compliance scan..."
+          return {}
```

Comment:
I'd honestly raise this as an exception. People will miss the log message and wonder why they're losing data.